### DISCUSSION TITLE
레벨5부터 방향성

### DISCUSSION THREAD

**Discussion by jminkkk** [woowacourse-teams/2024-code-zap#894] [이야기해봐요]
# 레벨5 방향성

- 정기회의
  - 매주 목요일 10시반 잠실  (오프라인)
  - 매주 월요일 2시 온라인

## 코드 리뷰 방식 변경

- 백엔드, 프론트엔드
  - 다음날 24시까지 무조건
  - 1명 이상 (백)

**Comment by jminkkk** [comment]
### 몰리

TPS 개선, 검색 속도 개선, 플러그인, 캐시 서버, 멀티모듈, 아키텍처 개선

### 켬미

PostgreSQL, 플러그인, 모니터링 URL 문제

### 제우스

새 기능: 문의하기(#851) 이메일, 관리자 페이지
리팩토링: 테스트코드에서 TestContainer를 사용하도록 변경(#840), HTTP Compression(#850)

### 짱수

OAuth(Github)

### 초롱

검색 속도 개선, 댓글 기능, 테스트 환경

**Comment by kyum-q** [comment]
## 7차 스프린트 최종 목표

### UI/UX 개선
- 템플릿 저장 후 로딩 피드백 구현
- 카테고리 생성 및 조회 하는 부분이 더 자연스럽게 구현
- 404 아닐 때 에러 페이지
- 프론트 에러 핸들링
- 프론트 테스트 환경 개선

### 기능 구현
- QueryDSL로 변경
- OAuth
- AWS 마이그레이션
- 파일명 선택으로 변경
  - 파일명 없으면 : 랜덤 문자열.txt
- 문의하기 API 생성 
- 배너(인기 태그) 캐시

### 버그 
- 모니터링 슬랙 알림 url 문제 해결
- 검색 속도 개선
   - Total Page 제거 (템플릿 검색, 좋아요한 템플릿)

### 도전 (완료는 어려울 수도)
- 인텔리제이 플러그인
- 퍼널 분리
- 관리자 페이지
  - 게시글 삭제
